lib/trino-filesystem/src/main/java/io/trino/filesystem/local/LocalInputFile.java

```diff
@@ -38,6 +40,13 @@ public TrinoInput newInput()
         return new LocalInput(file);
     }
 
+    @Override
+    public SeekableInputStream newStream()
+            throws IOException
+    {
+        return new FileSeekableInputStream(new RandomAccessFile(file, "r"));
```

Comment:
It is a bit inconsistent that `LocalInput` takes a file an `FileSeekableInputStream` takes a `RandomAccessFile`.  I'm fine if you want to leave this... just pointing out the inconsistency